src/test/java/com/lsd/core/report/approval/LsdContextTest.java

```diff
@@ -119,7 +119,25 @@ void notesCanBeAddedToParticipants() {
 
         Approvals.verifyHtml(lsdContext.generateReport("Adding notes to participants"));
     }
-    
+
+    @Test
+    void participantAreColoured() {
+        var arnie = ACTOR.called("Arnie", null, "#green");
+        var bettie = BOUNDARY.called("Bettie", null, "#red");
+        var cat = CONTROL.called("Cat", null, "#blue");
```

Comment:
As mentioned we could remove the `#` from the colour when added to the markup.

Reply:
Done